config/default/kustomization.yaml

```diff
@@ -11,7 +11,7 @@ bases:
 - ../manager
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
-#- ../webhook
+- ../webhook
```

Comment:
make bundle is broken right now:
`ERRO[0000] Error: Value ptp-operator: invalid service account found in bundle. sa name cannot match service account defined for deployment spec in CSV`
I will look at fixing this in a separate PR, because it's unrelated.
make manifests is run.